### DISCUSSION TITLE
Run The Repl To see output

### DISCUSSION THREAD

**Discussion by jkemp814** [iptv-org/community#241] [Q&A] [answered]
@LaneSh4d0w what does this mean? There is nothing to select.

https://i.imgur.com/GZ9aXm8.png

**Comment by BellezaEmporium** [comment, marked as answer]
You might have either got a link to a repo that's not launched / not available or simply deleted.

(if you're referring to query-streamlink, links have changed, it's now https://query-streamlink.herokuapp.com)

**Comment by jkemp814** [comment]
Line: 5550 in us.m3u playlist.

I went to this link: https://iptv-all.lanesh4d0w.repl.co/united-states/stockchartstv

**Reply by BellezaEmporium** [reply]
Needs to be replaced to https://iptv-all.lanesh4d0w.codes/united-states/stockchartstv

Weird that I didn't catch that one.
